app/webpacker/components/CompetitionForm/FormSections/DuesEstimate.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Input, Modal } from 'semantic-ui-react';
+import { QueryClient, useQuery } from '@tanstack/react-query';
+import useInputState from '../../../lib/hooks/useInputState';
+import { calculateDuesUrl } from '../../../lib/requests/routes.js.erb';
+import { currenciesData } from '../../../lib/wca-data.js.erb';
+import Loading from '../../Requests/Loading';
+import Errored from '../../Requests/Errored';
+import { fetchJsonOrError } from '../../../lib/requests/fetchWithAuthenticityToken';
+
+const CALCULATE_DUES_QUERY_CLIENT = new QueryClient();
```

Comment:
Usually we wrap the component in a query client. I don't really know what the difference is though.

Reply:
We wrap it everywhere we can. To my knowledge, there is one instance of a `const` client being passed around, which is `WcaSearch` (and `IdWcaSearch` in the same file). That's because these are form components where we cannot be sure in which contexts they are used, and under what circumstances a client may or may not be available.